config/file_config.go

```diff
@@ -368,7 +368,9 @@ func NewConfig(opts *CmdEnv, errorCallback func(error)) (Config, error) {
 	cfg.callbacks = make([]func(), 0)
 	cfg.errorCallback = errorCallback
 
-	go cfg.monitor()
+	if cfg.mainConfig.General.ConfigReloadInterval > 0 {
```

Comment:
Since we're not calling monitor any more, that means that f.ticker is nil, which means that if Stop() gets called it will panic. Please add a nil check to Stop().

Reply:
Nice catch